Sources/NIOEmbedded/AsyncTestingEventLoop.swift

```diff
@@ -171,6 +171,15 @@ public final class NIOAsyncTestingEventLoop: EventLoop, @unchecked Sendable {
         return self.scheduleTask(deadline: self.now + `in`, task)
     }
 
+    @discardableResult
+    public func scheduleCallback(in amount: TimeAmount, handler: some NIOScheduledCallbackHandler) -> NIOScheduledCallback {
+        // TODO: docs
+        /// Even though this type does not conform to `CustomTimerImplemenation`, it has a manual clock so we cannot
```

Comment:
Nit: the comment here and in `EmbeddedEventLoop` refer to a protocol that no longer exists.